### ISSUE TITLE
Unable to Fine Bridge In Hue App

### ISSUE THREAD

**Issue by ryancasler** [diyhue/diyHue#309]
I am unable to find my bridge in the Hue app.  I even loaded the default config.json file and was still unable to find it.  It flashes briefly that it found a bridge but then immediately changes to "no new bridges found".  Any assistance would be appreciated.

**Comment by Mevel** [comment]
Can you try hue essentials (android) to see if it is app or bridge related?!

**Comment by ryancasler** [comment]
I am able to find the bridge in Hue Essentials.  When I search in the Hue app, it pops up momentarily that it found a bridge, but then immediately changes to "no new bridges found".

**Comment by mariusmotea** [comment]
How many interfaces are connected on emulator host. I think the emulator provide the ip of second ibterface.

**Comment by ryancasler** [comment]
I can see the list on Hue Essentials but it doesn't allow me to edit it.  There are 4 copies of Hue Essentials listed, all from my phone.  My phone has a static IP on my network and I haven't tried to control it remotely, so I don't know why that has all that listed.  But how do I remove them?  If i choose delete it prompts me to log into "Away from Home" which I am.  It then does not remove the extra connections.  Is it safe to remove them directly from the json file?

**Comment by mariusmotea** [comment]
You can delete the users from whitelist as long as there where no rules or schedules created with that user. Start the emuator in debug mode to see if there are errors during pair process

**Comment by alexyao2015** [comment]
If you use the "beta" branch does it work?

**Comment by ryancasler** [comment]
No, it does not.  I just tried it.

**Comment by alexyao2015** [comment]
Well if that doesn't help, it definitely hasn't worked for a long time then because that hasn't been updated in while.

**Comment by ryancasler** [comment]
I ended up rebuilding the entire configuration fresh and that fixed the issue.

**Comment by alexyao2015** [comment]
The issue with this here is that there is some underlying bug that caused the config to break in the first place. If you had exact replication steps starting from a clean state to the broken state, it would make finding the problem easier.

**Comment by ryancasler** [comment]
The issue for finding the Hub was rebuilding the key file.  There were too many things I changed between when I setup deconz and when I finally noticed the inability to add new lights.  Existing lights worked for the whole time so I didn't really know this had happened till quite a bit later.  There were so many things in the intervening period I wouldn't know where to start.  

At this point, I can't think of anything specific that would be helpful.  However, if I am able to come up with anything, I will be sure to share it.  Thanks!!
